log4j-api/src/main/java/org/apache/logging/log4j/util/PropertiesUtil.java

```diff
@@ -23,6 +23,7 @@
 import java.io.InputStream;
 import java.nio.charset.Charset;
 import java.time.Duration;
+import java.time.format.DateTimeParseException;
```

Comment:
`DateTimeParseException` doesn't display any of its input arguments when printed. Could you instead use `IllegalArgumentException` with self-explanatory messages, please?

For instance, consider the following _example_:
```
final String message = String.format("provided duration `%s` doesn't match the expected `%s` pattern", input, pattern);
throw new IllegalArgumentException(message, cause);
```